services/env_vars.rb

```diff
@@ -0,0 +1,18 @@
+require 'dotenv'
+
+Dotenv.load
```

Comment:
You can do this like `require 'dotenv/load'`

Reply:
Great!!